content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -47,6 +47,11 @@ Then verify the blob by using `cosign`:
 cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert
 ```
 
+{{< note >}}
+cosign v1.9.0 is required to be able to use the `--certificate` flag.
+Please use `--cert` for older versions of cosign.
+{{< /note >}}
```

Comment:
I suggest making the following paragraph _not_ be a note. The page will read better as a result, and people will still see the link to learn more.

_Extra credit_: copy that link to a “what's next” section at the foot of the page (see other tasks pages for an example)

Reply:
I removed the `note`. Regarding the "What's next" section: What should we expect content wise there? Wouldn't we duplicate information if mentioning again that cosign has to be up to date? We could link to the cosign changelog and show how it can be updated, though.

Reply:
In docs, it's OK to duplicate information from earlier into the page into the What's Next section.

Something like this:
```markdown

## {{% heading "whatsnext" %}}

Read
[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
to learn more about Sigstore keyless signing.
```